doc/read-the-docs-site/quick-start.rst

```diff
@@ -0,0 +1,127 @@
+.. _quick_start:
```

Comment:
Shouldn't this be under `tutorials`? Since it is a tutorial.

Reply:
The quick start page is so important that I'd prefer it being a top-level page.